tests/api/v1/meals_api_endpoint.spec.js

```diff
@@ -0,0 +1,80 @@
+var shell = require('shelljs');
+var request = require('supertest');
+
+var Meal = require('../../../models').Meal;
+var Food = require('../../../models').Food;
```

Comment:
Hey are we using this?!!

Reply:
Probably not, I'll remove it.  Good catch. ⚾️